youtokentome/cpp/yttm.pyx

```diff
@@ -131,12 +132,22 @@ cdef class BPE:
         return subword.decode()
 
     def decode(self, ids, ignore_ids):
-        assert isinstance(ids, list)
-        assert isinstance(ignore_ids, list) or ignore_ids is None
+        assert isinstance(
```

Comment:
Suggest to replace this asserts on if statements plus raising of appropriate exception.

Reply:
Fixed